src/components/LocaleToggle/__test__/LocaleToggle.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { describe, expect, it } from 'vitest'
+import { render } from '@/tests/customRender'
+import LocaleToggle from '../LocaleToggle';
+import userEvent from '@testing-library/user-event'
+
+describe('LocaleToggle', () => {
+    it('renders correctly', () => {
+        const { getByText } = render(<LocaleToggle />);
```

Comment:
Let's use the [testing-library](https://testing-library.com/docs/react-testing-library/cheatsheet) style matchers for a better simulation of what the user experiences 

```suggestion
        const english = await screen.findByText('English')
        expect(english).toBeVisible()
```

You'll need to add `import { screen } from '@testing-library/react'` at the top

Reply:
Thank you! I have made the changes to use screen in both tests